.github/workflows/publish_image.yml

```diff
@@ -100,16 +106,14 @@ jobs:
            uses: docker/login-action@v3
            with:
              registry: quay.io
-             username: ${{ secrets.quayUser }}
-             password: ${{ secrets.quayPass }}
+             username: ${{ secrets.quayUser || secrets.QUAY_IO_USERNAME }}
+             password: ${{ secrets.quayPass || secrets.QUAY_IO_PASSWORD }}
```

Comment:
We always pass the `secrets.*` values from the Infinispan/Infinispan repository. Are you adding the `OR` logic for testing in isolation?

Reply:
yeah, while workflow_call supports secrets, workflow_dispatch doesn't and the suggested way is to have them defined as repo secrets